QualityControl/docs/Installation.md

```diff
@@ -0,0 +1,74 @@
+# Installation
+
+Follow these steps to install and set up QCG (Web UI for O<sup>2</sup> Quality Control).
+
+## 1. Install Node.js (version 22.x or higher)
+
+To run the application, you'll need Node.js version `>= 22.x`. Choose your operating system below to install Node.js:
+
+- **On CentOS 7 (CC7):**
```

Comment:
We are not supporting CC7 anymore and moved our production and supporting version to RHEL9. Moreover, the link installs nodejs version 16.9 which we do not support either